package.json

```diff
@@ -16,13 +16,14 @@
   "author": "Mhe Mrg <mhemrg120@gmail.com> (http://liara.ir/)",
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-backpack": "^0.4.3",
+    "@babel/core": "^7.4.3",
```

Comment:
Please don't upgrade npm packages unless you're sure that the project is compatible with them.
Currently, the CLI doesn't work with these new versions.

Reply:
with  the last config of `bable-core` which was version 6 is not compatible with `babel-loader` and I face to an error a lot,
now it's compatible with new version of `babel-core`, and I ran test suite and it was okay.